bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/UpnpDiscoveryService.java

```diff
@@ -102,6 +104,7 @@ protected void startScan() {
         }
         upnpService.getRegistry().addListener(this);
         upnpService.getControlPoint().search();
+        upnpService.getControlPoint().search(new RootDeviceHeader());
```

Comment:
As this is in the framework class and not within the WeMo binding: Could you comment a bit on the possible impact here? Can this potentially duplicate the amount of UPnP messages in the network? Is this something that should be done / makes sense according to the UPnP spec?

Is the discovery also solved for the "background" mode, where the discovery is not manually triggered (and thus this code is not run)? I.e. is a WeMo that is put into a socket and announces itself, discovered correctly?